e2e/e2e_test.go

```diff
@@ -446,22 +445,22 @@ func (c *ClientInfo) deleteReplicaSet(replicaSet *apps.ReplicaSet) error {
 	return nil
 }
 
-func (v envVars) maxReplicas(allPods []v1.Pod) int32 {
+func (v envVars) maxReplicas(allPods []core.Pod) int32 {
 	const maxPodsPerNode = 110
 	return int32(
 		(v.numComputeNodes*maxPodsPerNode - (len(allPods))) * v.fillPercentCapacity / 100)
 }
 
 // Waits for all replicas to be fully removed from replicaset, and checks that there are 0 ip pool allocations
-func checkZeroIPPoolAllocationsAndReplicas(replicaSet *apps.ReplicaSet, clientInfo *ClientInfo, k8sIPAM *kubeClient.KubernetesIPAM) error {
+func checkZeroIPPoolAllocationsAndReplicas(clientInfo *ClientInfo, k8sIPAM *kubeClient.KubernetesIPAM) error {
 	const (
 		emptyReplicaSet = 0
 		rsSteadyTimeout = 2 * rsCreateTimeout
 	)
 	var err error
 
 	// Set to 0 replicas
-	replicaSet, err = clientInfo.updateReplicaSet(replicaSetObject(
+	replicaSet, err := clientInfo.updateReplicaSet(replicaSetObject(
 		emptyReplicaSet,
 		podTierLabel(rsName),
```

Comment:
this is basically using a constant defined in https://github.com/k8snetworkplumbingwg/whereabouts/pull/181/commits/f9481dded3a92dec83d26fbefafdd0e3dc2f3506#diff-e91d29c6d4c58503d9c2d72e2b04f1b70901a613e23008d1d1cfe8beff7038d4R40

Please pass the name (and namespace) of the replicaset as an input parameter. We want this to work for more than one scenario.